.coderabbit.yaml

```
@@ -0,0 +1,242 @@
+        
+  # Security-focused unit test generation
+  unit_tests:
+    path_instructions:
+      - path: "src/server/**"
+        instructions: >
+          Generate tests covering security scenarios, authentication edge
+          cases, input validation, and error conditions
+      - path: "src/app/**"
+        instructions: >
+          Generate component tests covering user interactions, accessibility,
+          and error states
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)